src/common/zx-enhance.ts

```diff
@@ -53,13 +54,13 @@ export const ask = async (query: string, options?: AskType): Promise<string> =>
   const defaultAnswer = options?.defaultAnswer ?? ''
   const maxRetries = options?.maxRetries ?? MAX_RETRIES_QUESTION
```

Comment:
why use a `MAX_RETRIES_QUESTION`?

Reply:
Because if it is not limited, it can land in an infinite loop, that can only be exited through CTRL/CMD+C, not a nice shutdown.
By limiting the maximum of retries, it can catch it, throw an error and exit "cleanly"

If you are asking why using the `MAX_RETRIES_QUESTION` constant specifically, as a default value, can be replaced with a hard-coded number, if you so prefer.